sriov/tests/common/test_utils.py

```diff
@@ -26,17 +26,6 @@
 )
 
 
-def test_get_pci_address(dut, settings):
```

Comment:
Note that there is no real way I could think of to test this now. There is no field in the yaml for PCI, so there is nothing to check against. Perhaps another Linux command? That's still making assumptions, so perhaps we don't have this test.

Reply:
@dkosteck Take a look of how this test was done in PR#145 and let me know what you think